src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/application/CreateApplicationDto.kt

```diff
@@ -35,4 +36,7 @@ data class CreateApplicationDto(
   var visitorSupport: ApplicationSupportDto? = null,
   @Schema(description = "Username for user who actioned this request", required = true)
```

Comment:
will this need to change based on userType?

Reply:
The actionedBy will use the booker reference sent down from font end,... the orchestration now deals with it..